Add page-types for guide areas of MDN

teoli2003 opened the pull request:
This is a long tail of scattered guides on MDN.

I used `guide` and `landing-pages`.

Note: Under the PWA section, there are a few pages that match the modern docs structures. So I created the page types `howto` and `tutorial`, but used them ONLY for places where the 4-types-of-doc structure have been used.

This is part of openwebdocs/project#91

teoli2003 committed:
Add page-types for guide areas of MDN

wbamberg commented on files/en-us/web/progressive_web_apps/how_to/associate_files_with_your_pwa/index.md:
```diff
@@ -1,6 +1,7 @@
 ---
 title: Associate files with your PWA
 slug: Web/Progressive_web_apps/How_to/Associate_files_with_your_PWA
+page-type: howto
```
`how-to` seems easier to read than `howto`, is there a reason for this choice?

teoli2003 replied:
Fixed

teoli2003 committed:
howto -> how-to

teoli2003 committed:
Add tutorial-chapter

teoli2003 commented on files/en-us/web/progressive_web_apps/tutorials/intro/html_and_css/index.md:
```diff
@@ -1,6 +1,7 @@
 ---
 title: "PWA tutorial: Base HTML and CSS"
 slug: Web/Progressive_web_apps/Tutorials/Intro/HTML_and_CSS
+page-type: tutorial
```
Makes sense

teoli2003 committed:
Merge with main

teoli2003 commented:
So, I have:
- merged with main
- renamed `howto` to `how-to`
- renamed some `tutorial` into `tutorial-chapter`

I think this is read for a second review.

teoli2003 committed:
Merge

Elchi3 reviewed (approved):
Thanks Jean-Yves! I really like the idea of using "how-to" and "tutorial" for newly created structures that follow divio.
